### ISSUE TITLE
Instructions how to install using antigen

### ISSUE THREAD

**Issue by chodorowicz** [g-plane/pnpm-shell-completion#7]
Do you have instructions/tips on installing this plugin with [antigen zsh plugin manager](https://github.com/zsh-users/antigen)?

**Comment by g-plane** [comment]
No, but I think it's similar to zinit/zplug so you can try it and PR is welcomed if you found how to do.

**Comment by wilhen01** [comment]
I've succeeded in installing it via `antigen bundle g-plane/pnpm-shell-completion@main` in my `.zshrc` file but it's not actually working 🤔 

@g-plane do you have any tips on debugging? I suspect either it hasn't installed properly or it's conflicting with some other shell-completion I have installed...

**Comment by g-plane** [comment]
Is it loaded successfully?

**Comment by wilhen01** [comment]
> Is it loaded successfully?

How can you tell? The antigen bundle command has run successfully but I'm not sure how to check if the plugin has actually loaded properly in zsh

**Comment by g-plane** [comment]
Does antigen provide any commands for debugging?

**Comment by wilhen01** [comment]
I'm not sure. I'll have to have a look when I next get some spare time...

**Comment by chodorowicz** [comment]
So, one problem I've faced were issues `compdef` `compinit` which were not loaded
```
/https-COLON--SLASH--SLASH-github.com-SLASH-g-plane-SLASH-pnpm-shell-completion/pnpm-shell-completion.plugin.zsh:91: command not found: compdef
```

It seems loading plugin with defer solves the issue

```
# .zsh_plugins.txt
g-plane/pnpm-shell-completion kind:defer
```

But after that another issue I'm getting is 
```
pnpm --filter_pnpm:17: no such file or directory: /Users/cho/Library/Caches/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-g-plane-SLASH-pnpm-shell-completion/pnpm-shell-completion
```

I guess the the magic which happens in `install.zsh` is triggered by antidote? 🤔

Anyway, I've moved to `zplug` and all works fine.

**Comment by sweethuman** [comment]
It works to just `zplug.zsh` inside the plugin directory. Or follow the manual steps.
